src/Common/src/System/Drawing/ColorTranslator.cs

```diff
@@ -248,11 +260,7 @@ public static Color FromHtml(string htmlColor)
                     InitializeHtmlSysColorTable();
                 }
 
-                object o = s_htmlSysColorTable[htmlColor.ToLower(CultureInfo.InvariantCulture)];
-                if (o != null)
-                {
-                    c = (Color)o;
-                }
+                s_htmlSysColorTable.TryGetValue(htmlColor.ToLower(CultureInfo.InvariantCulture), out c);
```

Comment:
Couldn't this cause a null ref in the next check of `c.IsEmpty`? If the color is not found on the dictionary, c is going to be null.

Reply:
Never mind, color is a struct 😄